projects/unix/Makefile

```diff
@@ -144,7 +144,7 @@ endif
 # base CFLAGS, LDLIBS, and LDFLAGS
 OPTFLAGS ?= -O3 -flto
 WARNFLAGS ?= -Wall
-CFLAGS += -ffast-math -fno-strict-aliasing -fvisibility=hidden -I../../src -I../../src/asm_defines -DM64P_PARALLEL
+CFLAGS += -fno-strict-aliasing -fvisibility=hidden -I../../src -I../../src/asm_defines -DM64P_PARALLEL
```

Comment:
I _suspect_ we could find a dedicated flag fixing this problem: https://scicomp.stackexchange.com/questions/20867/what-does-ffast-math-do

Disabling `fast-math` disable a lot of thing, maybe one or two specific flags would be enough.

Reply:
Yes:
> It enables -fno-trapping-math, -funsafe-math-optimizations, -ffinite-math-only, -fno-errno-math, -fno-signaling-nans, -fno-rounding-math, -fcx-limited-range and -fno-signed-zeros. Each of these flags violates IEEE in a different way

The N64 follows the IEEE standard as far as I can tell. For instance, -fno-rounding-math disables all rounding modes except "round to nearest", so all that code we have to switch between the 4 rounding modes is meaningless. I think case I think the NaN comparison was probably broken by -ffinite-math-only, but I think we should be emulating the N64 as close as possible, not fixing game specific issues 1-by-1 (when it can be avoided)